### PR TITLE
azuresql-config-added

### LINKED ISSUE #2540: Ingestion Connector - Azure SQL
Creating an ingestion source connector for Azure SQL that fetches the metadata from Azure SQL and ingests the same in OpenMetadata

### PR DESCRIPTION
**Pull request by codingwithabhi**
### Describe your changes :

I worked on the adding Azure SQL connector in ingestion.
fix #2540 

#
### Type of change :

- [x] New feature

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.

#
### Reviewers
@ayush-shah

### CODE REVIEW

File: ingestion/src/metadata/ingestion/source/azure_sql.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    host_port = "localhost:1433"
+    scheme = "mssql+pyodbc"
+    service_type = "AZURE SQL"
+    use_pyodbc: bool = True
+    uri_string: str = ""
+
+    def get_connection_url(self):
+        if self.use_pyodbc:
+            self.scheme = "mssql+pyodbc"
+            return f"{self.scheme}://{self.uri_string}"
```
Comment (ayush-shah) [inline ingestion/src/metadata/ingestion/source/azure_sql.py:30]: We are using pyodbc as default, we won't need a check here in get_connection_url

**Comment by ayush-shah** [comment]
@codingwithabhi can you add the config requirements in setup.py for users to install and add the service type in database services (remove the space).

**Comment by ayush-shah** [comment]
Hey @codingwithabhi this PR contains 2 azure sql files, can you keep the one which has latest updates.

**Comment by harshach** [comment]
@codingwithabhi need a rebase to merge conflicts above

**Review by ayush-shah — APPROVED** [review]

**Pull request merged**